enhancements/machine-api/remediation-history.md

```diff
@@ -26,7 +26,7 @@ status: implementable
 
 ## Summary
 
-Record and show limited remediation history.
+Record and show limited remediation history in baremetal environments.
```

Comment:
I see this update 👀  Does this mean it won't be recorded in non baremetal environments?

Reply:
Oh, I think I misunderstood an earlier comment. Will remove it, thanks.